### ISSUE TITLE
The site title in the header should *not* be a `<h1>`.

### ISSUE THREAD

**Issue by dominikwilkowski** [govau/uikit#138]
Generally it's considered bad practice to have these as headings. There are however, some differing opinions. The [W3 says](https://www.w3.org/WAI/tutorials/page-structure/headings/) (emphasis mine):

> When looking at the content, the highest available heading level should be used to mark up the **heading of the main content**, as this makes it easy to discover.

Some of the confusion may come from the [module doco](http://uikit.apps.staging.digital.gov.au/packages/header/tests/site/) saying 'page title' not 'site title'.

When simply changing this from a heading, it doesn't look the same.

Do you want a separate bug/PR or is this feature still a work in progress?
